spec/charts/count_chart/count_chart_spec.rb

```diff
@@ -41,17 +41,19 @@
   describe '#default_options' do
     let(:chart) { Charts::CountChart.new [1] }
     it 'has a default item-colors' do
-      expect(chart.colors).to eq([
-                                   '#e41a1d',
-                                   '#377eb9',
-                                   '#4daf4b',
-                                   '#984ea4',
-                                   '#ff7f01',
-                                   '#ffff34',
-                                   '#a65629',
-                                   '#f781c0',
-                                   '#888888'
-                                 ])
+      expect(chart.colors).to eq(
+        [
+          '#e41a1d',
+          '#377eb9',
+          '#4daf4b',
+          '#984ea4',
+          '#ff7f01',
+          '#ffff34',
+          '#a65629',
+          '#f781c0',
+          '#888888'
+        ]
+      )
```

Comment:
This is good for longer lists.  Compare to the other shorter lists, and determine when you want to do this.